.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
The new files in this directory seem unrelated to this PR, and they seem to be causing pre-commit failures. Did they get added by accident?

Reply:
Sorry, my ide has generated this directory!